hackathon-website/src/Components/Parteneri/Parteneri.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import "./Parteneri.css";
+
+const parteneri = () => (
```

Comment:
Language consistency again.
Din moment ce majoritatea componentelor le aveti in engleza, as sugera sa pastrati consistenta si sa traduci din "parteneri" in "Partners".

Also, exista o conventie ca orice componenta care nu e un fel de wrapper, sa inceapa cu litera mare.

Reply:
Yep, un wrapper class e o componenta care adauga ceva in plus unei alte componente. Functioneaza exact ca si un Wrapper Design Pattern. Se mai numesc si Higher-Order components, si sunt acoperite in curs. 

Aici explica mai bine cum functioneaza:
https://reactjs.org/docs/higher-order-components.html

Reply:
Ioi, acuma inteleg. Parcursesem hco-urile. Multuu.